verification/curator-service/ui/cypress/integration/components/LinelistTableTest.spec.ts

```diff
@@ -15,4 +16,24 @@ describe('Linelist table', function () {
         cy.get('button[title="Save"]').click();
         cy.contains('test notes');
     })
+
+    it('Can edit a case', function () {
```

Comment:
nit: might be kind of helpful to arrange the tests in arrange/act/assert blocks, just for visual scanning purposes

Reply:
Done.